public/index.html

```diff
@@ -8,6 +8,7 @@
         <link rel="stylesheet" type="text/css" href="/assets/css/article.css">
         <link rel="stylesheet" type="text/css" href="/assets/css/header.css">
         <link rel="stylesheet" type="text/css" href="/assets/css/author.css">
+        <link rel="stylesheet" type="text/css" href="/assets/css/icons/rss.css">
```

Comment:
rebase and add this in the main.css

Reply:
Why main.css and not icon.css? Since it belongs to the folder "icons" and it is an icon